test-resources-plugin/src/main/java/io/micronaut/gradle/testresources/MicronautTestResourcesConsumerPlugin.java

```diff
@@ -44,7 +44,14 @@ public void apply(Project project) {
     }
 
     private Configuration createTestResourcesExtension(Project project) {
+        // Legacy configuration was only used in 3.5.0 so it's relatively safe
+        Configuration legacyConf = project.getConfigurations().create("testresources", conf -> {
```

Comment:
Perhaps a `TODO` to remember removing it in a future release?

Reply:
I'll create a ticket instead.

Reply:
https://github.com/micronaut-projects/micronaut-gradle-plugin/issues/509